core/node/state_keeper/src/metrics.rs

```diff
@@ -77,6 +91,32 @@ pub struct StateKeeperMetrics {
     pub blob_base_fee_too_high: Counter,
 }
 
+impl StateKeeperMetrics {
+    pub fn inc_rejected_txs(&self, reason: &'static str) {
+        self.tx_execution_result[&TxExecutionResult {
+            status: TxExecutionStatus::Rejected,
+            reason: Some(reason),
+        }]
+            .inc();
```

Comment:
Nit: Weird formatting here and below. I'd suggest to extract `TxExecutionResult` to a local var so that this issue doesn't repeat in the future.